tests/e2e-kubernetes/testsuites/util.go

```diff
@@ -76,6 +77,27 @@ func checkListingPath(f *framework.Framework, pod *v1.Pod, path string) {
 	e2evolume.VerifyExecInPodSucceed(f, pod, fmt.Sprintf("ls %s", path))
 }
 
+// checkBasicFileOperations verifies basic file operations works in the given `basePath` inside the `pod`.
+func checkBasicFileOperations(f *framework.Framework, pod *v1.Pod, basePath string) {
+	framework.Logf("Checking basic file operations inside pod %s at %s", pod.UID, basePath)
+
+	dir := filepath.Join(basePath, "test-dir")
+	first := filepath.Join(basePath, "first")
+	second := filepath.Join(dir, "second")
+
+	seed := time.Now().UTC().UnixNano()
+	testWriteSize := 1024 // 1KB
+
+	checkWriteToPath(f, pod, first, testWriteSize, seed)
+	checkReadFromPath(f, pod, first, testWriteSize, seed)
+	e2evolume.VerifyExecInPodSucceed(f, pod, fmt.Sprintf("mkdir %s && cd %s && touch %s", dir, dir, second))
+	checkListingPath(f, pod, dir)
```

Comment:
Is it worth verifying what the actual response is here?

Reply:
Addressed with https://github.com/awslabs/mountpoint-s3-csi-driver/pull/292/commits/fb140b96afaa2b20a09785fd55e3ef1d51bea5e3